### ISSUE TITLE
Radii Perturbation Equality

### ISSUE THREAD

**Issue by WilliamLanghoff** [WilliamLanghoff/hemo#3]
In the test for radii perturbations, due to flop arithmetic, volume before and after may not be equal. Should be testing that we are within a numerical tolerance (e.g. 10^-14?)

**Comment by 1riggs** [comment]
You're correct. I'm updating the test now, I'll create a branch and a pull request.
